packages/components/src/dialog/index.tsx

```diff
@@ -81,6 +83,7 @@ const Dialog = ( {
 			role="dialog"
 			shouldCloseOnEsc={ shouldCloseOnEsc }
 			shouldCloseOnOverlayClick={ shouldCloseOnOverlayClick }
+			bodyOpenClassName={ bodyOpenClassName }
```

Comment:
We expose the native [bodyOpenClassName](https://reactcommunity.org/react-modal/styles/classes/#for-the-documentbody-and-html-tag) property on the react-modal and make use of it for our CSS styling for the reauth modal.